frontend/test/metabase/scenarios/admin/troubleshooting/tasks.cy.spec.js

```diff
@@ -0,0 +1,53 @@
+import { signInAsAdmin, restore } from "__support__/cypress";
+
+describe("scenarios > admin > troubleshooting > tasks", () => {
+  beforeEach(() => {
+    restore();
+    signInAsAdmin();
+  });
+
+  it.skip("pagination should work (metabase#14636)", () => {
+    // Really ugly way to bump up total number of tasks (but more realistic than using fixture, or stubbing)
+    // Since this happens async, that number may vary but it should always be greater than 50 [1] and less than 100 [2]
+    // Note: each sync generates 6 tasks and we start with 12 tasks already for the testing sample dataset
+    for (let i = 0; i < 10; i++) {
+      cy.request("POST", "/api/database/1/sync_schema");
+    }
+    cy.server();
+    cy.route("GET", "/api/task?limit=50&offset=0").as("tasks");
+
+    cy.visit("/admin/troubleshooting/tasks");
+
+    cy.wait("@tasks").then(xhr => {
+      expect(xhr.response.body.total).to.be.greaterThan(50); /* [1] */
+      expect(xhr.response.body.total).to.be.lessThan(100); /* [2] */
+    });
+
+    cy.findByText("Troubleshooting logs");
+    cy.get(".Icon-chevronleft").as("previous");
+    cy.get(".Icon-chevronright").as("next");
+
+    cy.contains("1 - 50");
+    shouldBeDisabled("@previous");
+    shouldNotBeDisabled("@next");
+
+    cy.get("@next").click();
+    // 51 - any 2 digits number
+    cy.contains(/51 - \d{2}/);
+    cy.contains("1 - 50").should("not.exist");
+    shouldNotBeDisabled("@previous");
+    shouldBeDisabled("@next");
```

Comment:
I guess it's never going to create more than 100 tasks - if so, this would fail.

Reply:
It will not. My main concern is that it CAN createless than 50. Happened to me locally only once out of some 50 runs.
We have automatic test retries enabled so even if that happens, it should be sorted out in a second attempt.

Reply:
Hmm.... it's probably because the 10 syncs are fired too fast. I see three options - which one do you prefer? I don't love any of them, but probably 2 the least.
1. Add more iterations
2. Add a short wait between syncs
3. Stub the result

Reply:
I'll bump up sync requests, but like I said - **even if it fails**, the second rerun would be ok so test should pass in the end. We'd probably not even notice that it failed the first time.

Reply:
[20/20 tests passed](https://github.com/nemanjaglumac/metabase-tests/actions/runs/537379410) in a stress-test, so I'd rather not change anything here.
Merging